src/prototypecomponentsnavigatorwindow.h

```diff
@@ -0,0 +1,30 @@
+#ifndef PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+#define PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+
+#include <QDialog>
+class QStandardItemModel;
+class QItemSelectionModel;
+
+namespace Ui {
+class PrototypeComponentsNavigatorWindow;
+}
+
+class PrototypeComponentsNavigatorWindow : public QDialog
```

Comment:
It should not inherit from Dialog, as it is not dialog. It should behave as a regular window. Having QWidget as a base class should be enough.